.gitignore

```diff
@@ -2,7 +2,7 @@
 *.iml
 *.py[co]
 .*.sw[a-z]
-.coverage
+.coverage*
```

Comment:
looks like the names are like:
`.coverage.dev15-devc.1587745.424059`
so we can still ignore them.